extensions-contrib/opencensus-extensions/pom.xml

```diff
@@ -55,6 +55,36 @@
       <version>${project.parent.version}</version>
       <scope>provided</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.druid.extensions</groupId>
+      <artifactId>druid-kafka-indexing-service</artifactId>
+      <version>${project.parent.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>${apache.kafka.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>net.jpountz.lz4</groupId>
+          <artifactId>lz4</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
I'm curious why we had to exclude those, not a blocker, but it might be worth understanding

Reply:
I followed the pattern how kafka index extension did it. This change was introduced in this[ option to load classes from extension jars first PR](https://github.com/apache/druid/pull/5321/files) without further explanation.  For kafka index extension, it contains both lz4 (from parent pom dependency) and kafka-clients , and kafka-clients also contains lz4 which may leads to a version conflict when the kafka message is not compressed with lz4 (https://github.com/apache/druid/issues/3266).The same thing happens for slf4j. I can also find both `lz4` and `slf4j-api` dependency in parent pom.

Reply:
@xvrl In this case, shall we actually still exclude these two dependencies to avoid any dependency conflicts ?

Reply:
since we're only relying on kafka clients for very limited testing purposes, I don't think that's necessary